src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -81,6 +81,9 @@ extends AbstractMap[Key, Value]
     h ^ (h >>> 7) ^ (h >>> 4)
   }
 
+  /** Increase the size of the table.
+    * Copy only the occupied slots, effectively eliminating the deleted slots.
+    */
```

Comment:
I would also gently push for adding the explicit return type sooner rather than later, in case you make changes to this PR :)

Minor nitpick - we usually wrote docs like this in the past:

    /** First line.
     *  Second line (star indented 1 whitespace).
     */